src/main/java/me/kavin/piped/server/ServerLauncher.java

```diff
@@ -412,6 +412,14 @@ AsyncServlet mainServlet(Executor executor) {
                     } catch (Exception e) {
                         return getErrorResponse(e, request.getPath());
                     }
+                })).map(POST, "/user/playlists/clear", AsyncServlet.ofBlocking(executor, request -> {

```

Comment:
Please add a test for the in `./testing/api-test.sh`

Reply:
Done.
I also noticed that the test for 'rename playlist' use a wrong endpoint, I'll submit you a dedicated PR :wink:

Reply:
Oh, true, I wonder how nobody noticed that as tests should fail in theory because of that

Reply:
Looks like it didn't fail because we're not throwing errors with an actual status code in some places!